ArduPlane/quadplane.cpp

```diff
@@ -3924,6 +3924,16 @@ Vector2f QuadPlane::landing_desired_closing_velocity()
 
     // base target speed based on sqrt of distance
     float target_speed = safe_sqrt(2*transition_decel*dist);
+
+    // don't let the target speed go above landing approach speed
+    const float eas2tas = plane.ahrs.get_EAS2TAS();
+    float land_speed = plane.aparm.airspeed_cruise_cm * 0.01;
+    float tecs_land_airspeed = plane.TECS_controller.get_land_airspeed();
+    if (is_positive(tecs_land_airspeed)) {
+        land_speed = tecs_land_airspeed;
+    }
+    target_speed = MIN(target_speed, eas2tas * land_speed);
+
```

Comment:
As far as I can tell this never does anything, that is to say, we never actually set the speed anywhere. So I don't think the current code would ever cause us to speed up?

It is used as a threshold for switching into Pos1. Lowering it will make it more likely we will switch into Pos1 earlier.

Reply:
it is used in airbrake when TECS is still in charge, and can result in us adding throttle in airbrake mode